components/error_page/error_page.tsx

```diff
@@ -3,36 +3,39 @@
 
 import crypto from 'crypto';
 
-import PropTypes from 'prop-types';
 import React from 'react';
 import {FormattedMessage} from 'react-intl';
 import {Link} from 'react-router-dom';
 
 import {ErrorPageTypes, Constants} from 'utils/constants';
 import WarningIcon from 'components/widgets/icons/fa_warning_icon';
 
-import ErrorTitle from './error_title.jsx';
-import ErrorMessage from './error_message.jsx';
+import ErrorTitle from './error_title';
+import ErrorMessage from './error_message';
 
-export default class ErrorPage extends React.PureComponent {
-    static propTypes = {
-        location: PropTypes.object.isRequired,
-        asymmetricSigningPublicKey: PropTypes.string,
-        siteName: PropTypes.string,
-        isGuest: PropTypes.bool,
-    };
+type Location = {
+    search: string;
+}
+
+type Props = {
+    location: Location;
+    asymmetricSigningPublicKey?: string;
+    siteName?: string;
+    isGuest?: boolean;
+}
 
-    componentDidMount() {
+export default class ErrorPage extends React.PureComponent<Props> {
+    public componentDidMount() {
         document.body.setAttribute('class', 'sticky error');
     }
 
-    componentWillUnmount() {
+    public componentWillUnmount() {
```

Comment:
These `public` keywords are used because of the `explicit-member-accessibility` rule.